library/build.gradle

```diff
@@ -1,16 +1,18 @@
 plugins {
     id 'com.srcclr.gradle'
 
-    id 'com.gorylenko.gradle-git-properties' version '2.3.1'
-
     id 'bio.terra.pfb.java-library-conventions'
+    id 'maven-publish'
+    id 'com.jfrog.artifactory' version '5.0.3'
+    id 'com.gorylenko.gradle-git-properties' version '2.3.1'
```

Comment:
These three plugins are shared with the cli and library but it complains if I try to just set the version in buildSrc/build.gradle or the common convention file. @pshapiro4broad - Do you know how I should reference these plugins?

Reply:
I'm wondering if it's related to [this thread](https://github.com/DataBiosphere/java-pfb/pull/1#discussion_r1268267393).  @davidangb  - I don't think I've resolved this issue yet.

Reply:
This is resolved!